composer.json

```diff
@@ -22,6 +22,7 @@
         ]
     },
     "require": {
-        "phpoffice/phpspreadsheet": "^1.8"
+        "phpoffice/phpspreadsheet": "^1.8",
+        "friendsofphp/php-cs-fixer": "^2.16"
```

Comment:
we shouldnt need this as the github action only use the configuration file

Reply:
Right, thanks